tools/crs_converter/CRS_macros.xml

```diff
@@ -0,0 +1,140 @@
+<macros>
+    <xml name='ellps_options'>
+        <param name ='ellps' type='select' checked='true'
+               label='Ellipsoid and Datum' help='The name of built-in ellipsoid definition. 
+               See https://proj.org/en/9.4/usage/ellipsoids.html#ellipsoids for more informations.'>
+            <option value='GRS80'>GRS80</option>
+            <option value='airy'>airy</option>
+            <option value='bessel'>bessel</option>
+            <option value='clrk66'>clrk66</option>
+            <option value='WGS60'>WGS60</option>
+            <option value='WGS66'>WGS66</option>
+            <option value='WGS72'>WGS72</option>
+            <option value='WGS84' selected='true'>WGS84</option>
+            <option value='sphere'>sphere</option>
+        </param>
+    </xml>
+    
+    
+    <xml name='latitude_options'>
+        <param name ='lat_0' type='integer' value='0' 
+               label='Latitude' help='Latitude of natural origin, 
+               latitude of false origin or latitude of projection centre 
+               (naming and meaning depend on the projection method).'/>
+    </xml>
+    
+    
+    <xml name='latitude_1_options'>
+        <param name ='lat_1' type='integer' value='1' 
+               label='Latitude 1' help='First standard parallel.'/>
+    </xml>
+    
+    
+    <xml name='latitude_2_options'>
+        <param name ='lat_2' type='integer' value='1' 
+               label='Latitude 2' help='Second standard parallel.'/>
+    </xml>
+    
+    
+    <xml name='latitude_ts_options'>
+        <param name ='lat_ts' type='integer' value='0' 
+               label='Latitude of true scale' help='Defines the latitude where scale is not distorted. 
+               Takes precedence over scale factor K if both options are used together.'/>
+    </xml>
+    
+    
+    <xml name='longitude_options'>
+        <param name ='lon_0' type='integer' value='0' 
+               label='Longitude' help='Central meridian/longitude of natural origin, 
+               longitude of origin or longitude of false origin 
+               (naming and meaning depend on the projection method).'/>
+    </xml>
+    
+    
+    <xml name='X_options'>
+        <param name ='x_0' type='integer' value='0' 
+               label='X' help='False easting, easting at false origin or easting at projection centre 
+               (naming and meaning depend on the projection method). Always in meters.'/>
+    </xml>
+    
+    
+    <xml name='Y_options'>
+        <param name ='y_0' type='integer' value='0' 
+               label='Y' help='False northing, northing at false origin or northing at projection centre 
```

Comment:
I don't know much about projection and I don't understand this help part. Maybe it could be clearer for non expert

Reply:
I agree with you, however these descriptions are directly from the PROJ.org website, I'm not certain I can provide a bether description, I will try but I'm not expert myself!